### DISCUSSION TITLE
Regarding 2FA

### DISCUSSION THREAD

**Discussion by hd3291** [Velocidex/velociraptor#3530] [General]
Hi,

Is there a way to integrate 2FA on the user side of Velociraptor without SSO?

**Comment by scudette** [comment]
No - we only support oauth which delegates all authentication related things (like 2FA) to an oauth provider. There are a number of oauth providers that can be used to implement MFA and connectivity to more complex authentications scenarios - for example https://github.com/authelia/authelia  and maybe others.
